subnet/kube/kube.go

```diff
@@ -300,18 +270,28 @@ func nodeToLease(n v1.Node) (l subnet.Lease, err error) {
 	if err != nil {
 		return l, err
 	}
+
 	l.Attrs.BackendType = n.Annotations[backendTypeAnnotation]
 	l.Attrs.BackendData = json.RawMessage(n.Annotations[backendDataAnnotation])
+
 	_, cidr, err := net.ParseCIDR(n.Spec.PodCIDR)
 	if err != nil {
 		return l, err
 	}
+
 	l.Subnet = ip.FromIPNet(cidr)
-	l.Expiration = time.Now().Add(24 * time.Hour)
```

Comment:
Just thinking out loud here, would there be an impact of not specifying the time here? if you leave it blank it will take the `zero-value` for `time.Time` which is `0001-01-01 00:00:00 +0000 UTC`